### ISSUE TITLE
documentserver-configure.sh doesn't work with custom PostgreSQL database name

### ISSUE THREAD

**Issue by Soraphiroth** [ONLYOFFICE/DocumentServer#975]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
When using documentserver-configure.sh it keeps failing to connect to postgreSQL server. I checked the file and it seems that the parameters are incompletes:

```
establish_postgres_conn() {
	echo -n "Trying to establish PostgreSQL connection... "

	command -v psql >/dev/null 2>&1 || { echo "PostgreSQL client not found"; exit 1; }

        CONNECTION_PARAMS="-h$DB_HOST -U$DB_USER -w"
        if [ -n "$DB_PWD" ]; then
                export PGPASSWORD=$DB_PWD
        fi

        PSQL="psql -q $CONNECTION_PARAMS"
        CREATEDB="createdb $CONNECTION_PARAMS"

	$PSQL -c ";" >/dev/null 2>&1 || { echo "FAILURE"; exit 1; }

	echo "OK"
}
```
The database name is missing in `CONNECTION_PARAMS`. I edited it to `="-h$DB_HOST -d$DB_NAME -U$DB_USER -w"` and it worked.

**DocumentServer version:**
5.6.3

**Operating System:**
Centos 8

**Comment by ShockwaveNN** [comment]
Hi, am I right, what you use custom DB_NAME to conntect to postgres, because I've used default `onlyoffice` db for a long time in my test like this:

```
printf 'localhost\nonlyoffice\nonlyoffice\nonlyoffice\nlocalhost\nlocalhost\nguest\nguest\n' | sudo documentserver-configure.sh
```
and everything is fine

**Comment by ShockwaveNN** [comment]
Yeah, I was able to reproduce it
I've created an issue 46474 in our private issue tracker

**Comment by Soraphiroth** [comment]
Yes I'm using a custom name.

**Comment by ShockwaveNN** [comment]
This issue is fixed in https://github.com/ONLYOFFICE/document-server-package/pull/125 (sorry, private repo)

Fix will be available in future release v6.3.0 (currently no ETA)

**Comment by ShockwaveNN** [comment]
DocumentServer v6.3.0 is released. This problem should be fixed
